okhttp/src/main/java/okhttp3/WebSocketListener.kt

```diff
@@ -22,29 +22,35 @@ abstract class WebSocketListener {
    * Invoked when a web socket has been accepted by the remote peer and may begin transmitting
    * messages.
    */
-  open fun onOpen(webSocket: WebSocket, response: Response) {}
+  open fun onOpen(webSocket: WebSocket, response: Response) {
```

Comment:
we avoided `{}` in Java. Is it idiomatic in Kotlin?

Reply:
But don't make this your decision, pick a code formatter and apply e.g. ktlint :)